src/setupTests.js

```diff
@@ -1,33 +1,23 @@
-import React from 'react'
-import { act } from 'react-dom/test-utils'
 import Adapter from 'enzyme-adapter-react-16'
-import { configure, mount } from 'enzyme'
+import { configure } from 'enzyme'
 import 'jest-localstorage-mock'
+import './store'
+import { act } from 'react-dom/test-utils'
+import { initialize } from './initialize'
 
-import App, { initialized } from './App'
+/*******************************************************
+ *    Global setup that runs before each test file     *
+ * ****************************************************/
 
 configure({ adapter: new Adapter() })
 
-/** Dispatches an event on the window object. */
-export const windowEvent = (...args) =>
-  window.dispatchEvent(new KeyboardEvent(...args))
-
-/** Set up testing and mock document and window functions. */
-export const createTestApp = async () => {
-  await act(async () => {
-    jest.useFakeTimers()
-
-    // wait for app to be initialized
-    // specifically for initialSettings to be loaded via loadLocalState
-    await initialized
+beforeAll(async () => {
+  jest.useFakeTimers()
+  // initialize db and window events before every test
+  await initialize()
+})
```

Comment:
Unless I'm missing something, we still need to move `initialize` out into individual tests that need it, as we don't want it to run for every test. This is independent from mocking `store` per our previous discussion (#781).